sdk/cognitivelanguage/ai-language-text/CHANGELOG.md

```diff
@@ -1,24 +1,19 @@
 # Release History
 
-## 1.1.0-beta.1 (Unreleased)
+## 1.1.0-beta.1 (2022-11-17)
```

Comment:
is this assuming the classifications PR goes in tomorrow?

Reply:
Hopefully, but the JS client doesn't need it anyway because it renames `class` to `classifications` in all three classification types anyway.